libs/stats/odc/stats/_wofs.py

```diff
@@ -0,0 +1,105 @@
+"""
+Wofs Summary
+"""
+from typing import Optional
+import xarray as xr
+from odc.stats.model import Task
+from odc.algo.io import load_with_native_transform
+from odc.algo import safe_div, apply_numexpr
+from .model import OutputProduct, StatsPluginInterface
+from . import _plugins
+
+
+class StatsWofs(StatsPluginInterface):
+    def __init__(
+        self, resampling: str = "bilinear",
+    ):
+        self.resampling = resampling
+
+    def product(self, location: Optional[str] = None, **kw) -> OutputProduct:
+        name = "ga_s2_wo_summary"
+        short_name = "ga_s2_wo_summary"
+        version = "0.0.0"
+
+        if location is None:
+            bucket = "deafrica-stats-processing"  # TODO: ??
```

Comment:
If the location in None, it should throw an exception.

Reply:
yeah I just copied it across from previous one, what we really need is support a user supplied/default template .